coffee_cmd/src/coffee_term/command_show.rs

```diff
@@ -86,6 +89,49 @@ pub fn show_remote_list(remote_list: Result<CoffeeRemote, CoffeeError>) -> Resul
     Ok(())
 }
 
+pub fn show_nurse_verify(nurse_verify: &ChainOfResponsibilityStatus) -> Result<(), CoffeeError> {
+    if nurse_verify.defects.is_empty() {
+        term::success!("Coffee configuration is not corrupt! No need to run coffee nurse");
+    } else {
+        let mut table = radicle_term::Table::new(TableOptions::bordered());
+        table.push([
+            term::format::dim(String::from("●")),
+            term::format::bold(String::from("Defects")),
+            term::format::bold(String::from("Affected repositories")),
```

Comment:
In case the defect is `CoffeeGlobalRepoCleanup`, this column will be affected networks (not repositories) 
Maybe we could keep the message generic 
```suggestion
            term::format::bold(String::from("Affected")),
```

Reply:
I was thinking to move this label to action, what do you think?